types/is-gif/index.d.ts

```diff
@@ -1,12 +1,10 @@
-// Type definitions for is-gif 3.0
+// Type definitions for is-gif 4.0
 // Project: https://github.com/sindresorhus/is-gif#readme
 // Definitions by: Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 /// <reference types="node" />
 
 /**
- * Returns a boolean of whether input is a GIF image.
+ * Returns a boolean of whether `input` is a GIF image.
  */
-declare function isGif(input: Buffer | Uint8Array): boolean;
-
-export = isGif;
+export default function isGif(input: Buffer | Uint8Array): boolean;
```

Comment:
Buffer extends Uint8Array, so it might not be needed to be explicit and depend on node types.

Reply:
Yes, excellent remark (LSP), updated!
thx!